### ISSUE TITLE
Controller Uni Flensburg

### ISSUE THREAD

**Issue by FPienkny** [IEAWindTask37/IEA-3.4-130-RWT#3]
Hello Pietro,

I hope you're doing well! 

I have a question regarding the control of the wind turbine. Is there any documentation available for the Flensburg controller, preferably the uncompiled fortran file? As the NREL controller is not working very well around the rated wind velocity of the turbine I want to use the other one. However , the results of my simulations show some significant differences between the two controllers and I'd like to understand the behaviour of the controller better to identify the reasons for the differences.

Thank you very much in advance!

Best regards,
Friedrich

**Comment by ptrbortolotti** [comment]
Dear Friedrich, my apologies for the slow response. You may want to talk to David Schlipf about the Flensburg controller (david.schlipf@hs-flensburg.de), I'm lacking the details.
I'm however surprised to hear that the ROSCO controller is not behaving properly around rated. What do you mean by that?

**Comment by FPienkny** [comment]
Thanks! 

Actually, I tought the compiled NREL controller is the old baseline controller? However, it is responding very slowly and causes high load alternations (probably because it is jumping between the different regions?).

**Comment by ptrbortolotti** [comment]
You are totally right, my apologies. The controller folder has the old NREL controller adapted from the NREL 5MW.

I have not used the controller in ages and I'm not surprised that it does not work properly.

The OpenFAST model comes however with the inout files to the recently developed ROSCO controller
https://github.com/NREL/ROSCO/tree/master

I recommend compiling it and use that one

FYI, in the f/beamdyn branch I've just pushed the model compatible to the very latest OpenFAST (2.4.0) and generated by the latest version of WISDEM (3.0.3)

https://github.com/IEAWindTask37/IEA-3.4-130-RWT/tree/f/BeamDyn

Here I deleted the old controller files from NREL

I hope this helps

Pietro
